src/officehours_api/notifications.py

```diff
@@ -15,11 +15,14 @@
 twilio = TwilioClient(settings.TWILIO_ACCOUNT_SID, settings.TWILIO_AUTH_TOKEN)
 
 DOMAIN = Site.objects.get_current().domain
-PREF_URL = f"{DOMAIN}{reverse('preferences')}"
-ADDENDUM = (
-    f"\n\nYou opted in to receive these texts from U-M. "
-    f"Opt out at {PREF_URL}"
-)
+
+
+def build_addendum():
```

Comment:
Would it be worth adding a comment or docstring explaining why we did this?

Reply:
We both missed this initially, so someone else might not understand pick this up either.

Reply:
Good idea, added a comment.